app/src/main/java/ru/vssemikoz/newsfeed/models/NewsItem.java

```diff
@@ -11,7 +13,8 @@
     private String description;
     private String content;
     private URL url;
-    private URL imageUrl;
+    @SerializedName("urlToImage")
+    private URL imageUrl    ;
```

Comment:
Why do we need this tabbed space at the end of the line? BTW, does GSON works with this file without the default constructor (without any parameters)?